### DISCUSSION TITLE
Nextcloud/Troubleshooting/Collabora%20Code/

### DISCUSSION THREAD

**Discussion by giscus** [TrueBankai416/BankaiTechDocs#112] [Announcements]
# Nextcloud/Troubleshooting/Collabora%20Code/

502 Gateway errors when trying to connect to Collabora Code

https://docs.bankai-tech.com/Nextcloud/Troubleshooting/Collabora%20Code/

**Comment by TrueBankai416** [comment]
@dexterdy Ive been spending the last 3 days making major improvements to this site and your previous comment was removed because the file tree changed. I added it to this page (troubleshooting). Does this look alright to you?

**Reply by dexterdy** [reply]
This looks fine! Thanks for taking the time to save my comment
